src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -130,7 +126,8 @@ public void ValidPathWithTrailingSlash()
             });
         }
 
-        [Fact]
+        [ConditionalFact("UsingNewNormalization")]
```

Comment:
Nit: ```nameof(UsingNewNormalization)```
Could we also come up with a more descriptive name instead of "New"?  "New" will quickly lose its meaning.